### PR TITLE
kommander: Add Karma

### PR DESCRIPTION
**Pull request by branden**
https://jira.mesosphere.com/browse/DCOS-59076

This depends on #217.

This PR adds the `kommander-karma` chart from #217 as a subchart of `kommander`.

### CODE REVIEW

**Comment by branden** [comment]
@alejandroEsc This will be deployed by default when opsportal bumps kommander. That can be prevented by setting `kommander-karma.enabled=false`. LMK if you'd like me to submit a PR to add that to opsportal's values.

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -97,6 +97,10 @@ kommander-thanos:
   enabled: true
   thanosAddress: "prometheus-kubeaddons-prom-prometheus.kubeaddons.svc.cluster.local:10901"
 
+kommander-karma:
+  enabled: true
```
Comment (hectorj2f) [inline stable/kommander/values.yaml]: I am not sure if we want this enabled by default. I suggest you set it to false in kubeaddons-configs.

Comment (hectorj2f) [reply stable/kommander/values.yaml]: @branden ^

Comment (branden) [reply stable/kommander/values.yaml]: My understanding is that we want Karma enabled by default when Kommander is deployed.

Comment (branden) [reply stable/kommander/values.yaml]: /cc @gracedo @philipnrmn

Comment (gracedo) [reply stable/kommander/values.yaml]: That's also my understanding - that Karma should be enabled by default

**Pull request merged**